verification/scripts/README.md

```diff
@@ -0,0 +1,21 @@
+# Verification scripts
+
+This directory contains various scripts to manage the verification part of the epid system.
+
+## Bootstrapping admin roles
+
+When the application is first used, logged-in users have no roles associated with them.
+To bootstrap the admin process, you can assign a user with a given email the admin, reader and curator roles:
+
+```shell
+mongo "mongodb://user:pass@some-connection-string/somedb" \
```

Comment:
Would "user:pass@some-connection-string/somedb" need to be changed when run? I wouldn't know what to fill into those fields.

Reply:
For localhost (dev) this is "mongodb://localhost:27017/covid19" but for the atlas DB you have to use a proper connection string as the data service or curator services are doing on AWS. (you can go to the web atlas console and generate creds for you, they'll also show you the connection string to use those creds you just created)

Reply:
Could you update the instructions with that?

Reply:
certainly, done